filament/backend/test/test_Blit.cpp

```diff
@@ -269,6 +269,10 @@ TEST_F(BlitTest, ColorResolve) {
     SKIP_IF(Backend::WEBGPU, "test cases fail in WebGPU, see b/424157731");
     auto& api = getDriverApi();
 
+    if (api.getFeatureLevel() < FeatureLevel::FEATURE_LEVEL_2) {
+        GTEST_SKIP() << "Skipping test because multi-sampled textures are not supported at Feature level < 2";
+    }
```

Comment:
I wasn't aware of this.  But I guess vk and metal both have ms textures by default?

What does this mean in practice?  The client couldn't request msaa unless they also request FL2?